_examples/scatter/main.go

```diff
@@ -7,17 +7,23 @@ import (
 	_ "net/http/pprof"
 
 	"github.com/wcharczuk/go-chart"
+	"github.com/wcharczuk/go-chart/drawing"
 )
 
 func drawChart(res http.ResponseWriter, req *http.Request) {
+
+	viridisByY := func(xr, yr chart.Range, x, y float64) drawing.Color {
```

Comment:
Sure thing. Not sure what you mean yet but feel free to add it after the pull if it looks okay as a foundation to you.